### PR TITLE
06 time management

### PR DESCRIPTION
**Pull request by Happybolt**
1. Implement kernel module with API in sysfs which returns relation time.
1. Implement kernel module with API in sysfs which returns absolute time.
2. Implement kernel module with API in sysfs which return CPU load.

### CODE REVIEW

File: 06-TimeManagement/modul/time_management.c [hunk 1]
```diff
@@ -33,6 +33,27 @@ struct class_attribute attr_relation_time = {
...
 
+
+static ssize_t absolute_time_show(struct class *class,
+	struct class_attribute *attr, char *buf)
+{
+	static struct timespec prev_ts;
+
+	pr_info("TimingManagement:PrevAbsolute time: %lu.%lu\n", prev_ts.tv_sec,
+		prev_ts.tv_nsec);
+	sprintf(buf, "TimingManagement:PrevAbsolute time: %lu.%lu\n",
```
Comment (yekovalyov) [inline 06-TimeManagement/modul/time_management.c]: Error in format string.

File: 06-TimeManagement/modul/time_management.c [hunk 2]
```diff
@@ -33,6 +33,27 @@ struct class_attribute attr_relation_time = {
 	.show	= relation_time_show,
 };
 
+
+static ssize_t absolute_time_show(struct class *class,
+	struct class_attribute *attr, char *buf)
+{
+	static struct timespec prev_ts;
+
+	pr_info("TimingManagement:PrevAbsolute time: %lu.%lu\n", prev_ts.tv_sec,
```
Comment (yekovalyov) [inline 06-TimeManagement/modul/time_management.c]: Error in format string.

Comment (Happybolt) [reply 06-TimeManagement/modul/time_management.c]: Fixed: pr_info("TimingManagement:PrevAbsolute time: %lu.%09lu\n", prev_ts.tv_sec,
		prev_ts.tv_nsec);

File: 06-TimeManagement/modul/time_management.c (hunk 1 above)
Comment (Happybolt) [reply 06-TimeManagement/modul/time_management.c]: Fixed: sprintf(buf, "TimingManagement:PrevAbsolute time: %lu.%09lu\n",
		prev_ts.tv_sec, prev_ts.tv_nsec);

**Pull request merged**